### ISSUE TITLE
Does anybody has issues with ios 15?

### ISSUE THREAD

**Issue by elenkoooo** [praeclarum/sqlite-net#1105]
I have an issue when I try to connect to db (using both sync\async connection) on IOS 15+ I don't get an error code, only native stack trace (see below) Everything works perfect on IOS 14 and lower. Does anybody has the same problem?

Native stacktrace: 0x0 - Unknown 0x10a8a26ac - /Users/administrator/Library/Developer/CoreSimulator/Devices/F96EA4A8-4A2F-493B-8FF7-86EC96C77D2B/data/Containers/Bundle/Application/4E6AAB6E-C0B6-4BD5-ABC5-B0D9F19FFBC4/eDoc.iOS.app/eDoc.iOS : hookedSQLiteVFSOpen 0x7fff2204c7b0 - /Applications/Xcode_13.1.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libsqlite3.dylib : sqlite3BtreeOpen 0x7fff2202a2dc - /Applications/Xcode_13.1.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libsqlite3.dylib : openDatabase 0x1137440d1 - Unknown

I've tried to create text file at the same folder as database file - I don't get any error and I can read created file.

**Comment by xunreal75** [comment]
Yes, can't read data in Release mode on real device without error. All works fine in Simulator or Debug on iOS Device
